lib/northstar-endpoint.js

```diff
@@ -10,6 +12,47 @@ class NorthstarEndpoint {
     this.client = client;
   }
 
+  /**
+   * Helper function to execute simple get.
+   */
+  executeGet(endpoint, query) {
```

Comment:
Well, I wanted it to be generic in case we wanted to execute get or post out of scope.